rgd/utility.py

```diff
@@ -183,7 +187,8 @@ def patch_internal_presign(f: FieldFile):
     odd situation of accessing the file locally.
     """
     if (
-        isinstance(f.storage, MinioStorage)
+        MinioStorage is not None
```

Comment:
You really don't need this check at all. If `MINIO_STORAGE_MEDIA_URL` is defined, I think it's safe to assume that the storage provider is `MinioStorage `, particularly since the `MINIO_STORAGE_MEDIA_URL` setting is not defined (and thus can never be picked up even if the environment variable is set) in `ProductionConfig`: https://github.com/girder/django-composed-configuration/blob/master/composed_configuration/_configuration.py#L58